content/connect/index.textile

```diff
@@ -0,0 +1,227 @@
+---
+title: Connections
+meta_description: "Establish and maintain a persistent connection to Ably using the realtime interface of an Ably SDK."
+languages:
+  - javascript
+  - nodejs
+  - ruby
+  - java
+  - swift
+  - objc
+  - csharp
+  - flutter
+---
+
+The Ably realtime interface establishes and maintains a connection to the Ably service, using the most efficient transport available, typically "WebSockets":https://ably.com/topic/websockets, allowing you to monitor and manage "connection states":/connect/states. The Ably realtime interface operates and multiplexes all channel traffic over that connection. "Channels":/channels are separate from connections and are used to logically separate messages into different topics. 
+
+CAVEAT-INPUT
+
+  The Ably Realtime SDK opens and maintains a connection to the Ably realtime servers on instantiation, connections can only be established using the realtime interface of an Ably SDK. See "how to use Ably":/basics/use-ably for further information.
```

Comment:
I don't think we need the first sentence as it's a repeat of the intro and the paragraph below.

```suggestion
Connections can only be established using the realtime interface of an Ably SDK. See "how to use Ably":/basics/use-ably for further information.
```